app/src/main/assets/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+../../../../CHANGELOG.md
```

Comment:
is this a symlink?

Reply:
indeed

Reply:
This will work on Linux and MacOS, but not on WIndows.
Would that be a problem?